src/bundle/Resources/config/input_parsers.yml

```diff
@@ -159,6 +160,12 @@ services:
         tags:
             - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.FieldDefinitionUpdate }
 
+    ezpublish_rest.input.parser.JWTTokenInput:
+        parent: ezpublish_rest.input.parser
+        class: "%ezpublish_rest.input.parser.JWTTokenInput.class%"
```

Comment:
Please drop `class` attr and use FQCN as service name in all new service definitions.